api/src/main/java/ca/bc/gov/educ/eas/api/endpoint/v1/PENRecordEndpoint.java

```diff
@@ -0,0 +1,25 @@
+package ca.bc.gov.educ.eas.api.endpoint.v1;
+
+import ca.bc.gov.educ.eas.api.constants.v1.URL;
+import ca.bc.gov.educ.eas.api.struct.v1.StudentMerge;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+
+@RequestMapping(URL.BASE_URL + "/pen-records/createDateInRange")
```

Comment:
The path identifier createDateInRange could be moved to specific endpoint. Better to have in lowercase.

Reply:
Please modify this to:
`@RequestMapping(URL.BASE_URL + "/student-merges")`